cmd/minishift/cmd/start.go

```diff
@@ -409,26 +409,49 @@ func exportContainerImages(hostVm *host.Host, version string) {
 	fmt.Println(fmt.Sprintf("-- Exporting of OpenShift images is occuring in background process with pid %d.", exportCmd.Process.Pid))
 }
 
-// calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
-func calculateDiskSizeInMB(humanReadableDiskSize string) int {
-	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
+func calculateMemorySize(memorySize string) int {
+	if stringUtils.HasOnlyNumbers(memorySize) {
+		memorySize += "M"
```

Comment:
This is because of how Docker machine takes the value. With RAMInBytes` (which takes MiB values) which you suggested to try on IRC, you need to divide with MiB. The previous version used HumanSize, which does assume a unit, and therefore divides by MB.

Note: MiB can not be enforced as this does not pass the sizeregex, so mentioning it as MB, would still result in Mebibytes. Confused about your reasoning now...